manim/mobject/mobject.py

```diff
@@ -141,6 +141,9 @@ def add(self, *mobjects):
         self.submobjects = list_update(self.submobjects, mobjects)
         return self
 
+    def __add__(self, mobject):
+        return self.add(mobject)
+
```

Comment:
Question: what is the result of this code?
```python
g = VGroup(obj1, obj2)
(g + obj3).shift(RIGHT)
```

Under the current implementation, I believe this does three things: it shifts `g` to the right, it shifts `obj3` to the right, _and it also adds obj3 to g_. Is this correct? If so, I would find it a bit confusing that such code would modify the contents of `g`.

Reply:
This does modify g, but that change is in the other file https://github.com/ManimCommunity/manim/pull/790/files#diff-3f967d53f766c19d36e76e161c5d1ac70e25e4f789415cd92615424981225f22R1004